docs/HOWTO.md

```diff
@@ -171,3 +178,38 @@ claude mcp add azure-devops -- npx -y @azure-devops/mcp Contoso
 ```
 
 Replace `Contoso` with your own organization name
+
+# 🍇 Using MCP Server with Cursor
+
+To integrate the Azure DevOps MCP Server with Cursor, create a `.cursor\mcp.json` file and add your Azure DevOps organization to the `mcpServers` list.
+
+```json
+{
+  "mcpServers": {
+    "ado": {
+      "command": "npx",
+      "args": ["-y", "@azure-devops/mcp", "Contoso"]
```

Comment:
maybe wrap `Contoso` with braces to clarify this is a dummy org name and should be replaced with the actual org name?

```suggestion
      "args": ["-y", "@azure-devops/mcp", "{Contoso}"]
```